engine/updategraph/src/main/java/io/deephaven/engine/updategraph/UpdateGraphProcessor.java

```diff
@@ -1513,7 +1569,12 @@ private void refreshTablesAndFlushNotifications() {
         waitForNextCycle(startTime, sched);
     }
 
-    private void logCycle(final long cycleTimeNanos) {
+    private void computeStatsAndLogCycle(final long cycleTimeNanos) {
+        final long safePointPauseTimeMillis = jvmIntrospectionContext.deltaSafePointPausesTimeMillis();
+        accumulatedCycleStats.accumulate(
+                DEFAULT_TARGET_CYCLE_DURATION_MILLIS,
```

Comment:
Why would you use the default instead of the actual value?  That makes it even less worth computing the on/off budget here instead of later.

Reply:
See comment above.